GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -141,6 +143,36 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
             running = installed && gvfsService.IsRunning;
         }
 
+        public override bool TryCopyPanicLogs(string copyToDir, out string error)
+        {
+            error = null;
+            try
+            {
+                if (!Directory.Exists(DiagnosticReportsDirectory))
+                {
+                    return true;
+                }
+
+                string copyToPanicDir = Path.Combine(copyToDir, ProjFSKext.DriverLogDirectory, "panic_logs");
+                Directory.CreateDirectory(copyToPanicDir);
+
+                foreach (string filePath in Directory.GetFiles(DiagnosticReportsDirectory, PanicFileExtention))
+                {
+                    if (File.ReadAllText(filePath).Contains(ProjFSKext.DriverName))

```

Comment:
I think this will match any panics for which the ProjFS kext was *loaded*, not just those where it appeared in the stack trace. (There is a list of loaded kexts at the end of each panic log.) Is that the intended behaviour? Either way, it might be worth commenting on the *intention* of this check here.

Reply:
Nice catch!

> Is that the intended behaviour?

I think the intended behavior is to only include the panics where ProjFS kext appeared in the stack trace.

Reply:
Looking at several panics it appears that if we're in the backtrace, we'll have the form `org.vfsforgit.PrjFSKext(`. 

Would you advise that as the simplest way to do this?

Reply:
We discussed this out-of-band and this does seem like the simplest solution.